src/containers/Profile/index.js

```diff
@@ -155,10 +189,12 @@ export class Profile extends Component {
               <label htmlFor='phoneNumber'>
                 Phone Number
               </label>
+              <center style = {{color: 'red'}}> {this.state.phoneNumberError} </center>
```

Comment:
For consistency with how it's handled elsewhere, it would be preferable to handle the alignment and color styling through the CSS files rather than in-line.